serversettings.py

```diff
@@ -421,9 +421,9 @@ def saveAdvanced (self):
         if not preserve_job_history:
             job_history_line = "PreserveJobHistory No\n"
 
-        # Default is not to preserve job files.
+        # Default is to preserve job files.
         if preserve_job_files:
```

Comment:
@oubayashi ```preserve_job_files``` is True if the button is active -> PreserveJobFiles is 'Yes' or 'N' seconds. So the if condition is missing ```not``` IIUC.

Reply:
@oubayashi LGTM!